src/govuk/components/task-list/template.njk

```diff
@@ -0,0 +1,28 @@
+{% set idPrefix = params.idPrefix if params.idPrefix else "task-list" %}
+<ul class="govuk-task-list">
+  {% for item in params.items %}
+    {% if item.hint %}
+      {%- set hintId = idPrefix + "-" + loop.index + "-hint" -%}
+    {% endif %}
+    {%- set statusId = idPrefix + "-" + loop.index + "-status" -%}
+    <li class="govuk-task-list__item{% if item.href %} govuk-task-list__item--with-link{% endif %}">
+      <div class="govuk-task-list__task-name-and-hint">
+        {% if item.href %}
+          <a class="govuk-link govuk-task-list__link" href="{{ item.href }}" aria-describedby="{{ hintId + " " if item.hint }}{{ statusId }}">{{ item.title.html | safe if item.title.html else item.title.text }}</a>
+        {% else %}
+          <div class="govuk-task-list__task-no-link">{{ item.title.html | safe if item.title.html else item.title.text }}</div>
+        {% endif %}
+        {% if item.hint %}
+          <div id="{{ hintId }}" class="govuk-task-list__task_hint">
+            {{ item.hint.html | safe if item.hint.html else item.hint.text }}
+          </div>
+        {% endif %}
+      </div>
+      <div class="govuk-task-list__status-container">
+        <span id="{{ statusId }}" class="govuk-task-list__status {{ item.status.classes }}">
```

Comment:
@simonwhatley we debated this quite a bit. The thinking was that we wanted to deviate from the existing Tag component styles (ie uppercase text with bright blue background) as there was a fair amount of concern (and some research evidence) that the tag component looked too clickable when used within the task list pattern, as the design is quite close to the button design, and that the uppercase text may be less accessible.

There are a few different approaches we could take:

1. also update the styling of the Tag component at the same time 
2. use the Tag component but add some extra modifiers to completely change the style
3. not use the tag component at all, and instead treat it as a custom 'status' element of the Task list

We went with 3 mainly for pragmatism, to avoid having to make backwards-incompatible changes to the Tag component (which is also used by the Phase banner component). The second option feels a bit pointless to me, as we’d be changing so much of the styling it wouldn't share that much in common...?

It’s also worth noting that one of the styles for the status component is just plain black text (for "Complete") - which differs quite a lot from the Tag component.

The downside with option 3 though is that we’d have to live with some visual inconsistency between the Tag component and the status bit of the task list component, at least in the short term whilst any changes to the Tag component are debated...

See [previous comment on this](https://github.com/alphagov/govuk-frontend/pull/2261/files#r837395296).

Reply:
@philsherry yep - we discussed that design (and the blog posts) a fair bit!  It helped convince us to both make the statuses look less clickable AND to make them actually clickable. Belt and braces... 😄